CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to the PyScript CLI will be documented in this file, with the latest release at the top.
+
+## [0.2.6] - 2023-03-06
+
+*A lot has changed between 0.2.5 and 0.2.6. This is a summary of the most important changes.*
```

Comment:
if that's semver, doesn't this qualify for a minor bump if there's anything breaking at all? 🤔

Reply:
This is a minor bump right? from 0.2.5 to 0.2.6 the reason why i mention that a lot has changed is because it's been almost a year and half since the last release and a bunch of stuff went in 😆

Reply:
Oh wait... I think you meant we should do 0.3.0?  Sorry I'm tired haha

Reply:
Yes, I think that's what he meant and +1 from me

Reply:
Done thanks 😄